modules/ckanext-ytp_main/ckanext/ytp/templates/package/resource_read.html

```diff
@@ -30,11 +32,11 @@
               <ul>
                 {% block resource_actions_inner %}
                   {% if h.check_access('package_update', {'id':pkg.id }) %}
-                    <li>{% link_for _('Manage'), controller='package', action='resource_edit', id=pkg.name, resource_id=res.id, class_='btn', icon='wrench' %}</li>
+                    <li>{% link_for _('Manage'), controller='package', action='resource_edit', id=pkg.name, resource_id=res.id, class_='btn btn-menu-manage', icon='wrench' %}</li>
```

Comment:
```suggestion
                    <li>{% link_for _('Manage'), named_route=pkg.type ~ '_resource.edit', id=pkg.name, resource_id=res.id, class_='btn btn-menu-manage', icon='wrench' %}</li>
```
The controller/action link will break in CKAN 2.9. Use named routes